dockstore-client/src/test/java/io/dockstore/common/FileProvisionTest.java

```diff
@@ -1,10 +1,11 @@
 package io.dockstore.common;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Optional;
+import java.io.File;
+import java.net.URI;
+import java.util.*;
```

Comment:
Didn't notice that this got changed. Surprised stylecheck or maven clean install didn't catch this